### PR TITLE
ZMDI Icons suport

### PR DESCRIPTION
**Pull request by cirovargas**
Suport ZMDI icons from: http://zavoloklom.github.io/material-design-iconic-font/

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -213,7 +213,7 @@ protected function addFormConfig(ArrayNodeDefinition $rootNode)
 
     protected function addIconsConfig(ArrayNodeDefinition $rootNode)
     {
-        $iconSets = array('glyphicons', 'fontawesome', 'fontawesome4');
+        $iconSets = array('glyphicons', 'fontawesome', 'fontawesome4','zmdi');
```
Comment (isometriks) [inline DependencyInjection/Configuration.php]: Missing space before `'zmdi'`

**Comment by isometriks** [comment]
Please squash to one commit and I will merge. Thanks!

**Comment by cirovargas** [comment]
Ok

**Comment by isometriks** [comment]
You can leave this PR open and force push your branch after you squash

**Pull request closed without merging**